pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -207,20 +207,20 @@ void processArguments() throws Exception {
                 description = "Path to the main Go executable binary for the function (if the function is written in Go)")
         protected String goFile;
         @Parameter(names = {"-i",
-                "--inputs"}, description = "The function's input topic or topics (multiple topics can be specified as a comma-separated list)")
+                "--inputs"}, description = "The input topic or topics (multiple topics can be specified as a comma-separated list) of functions")
```

Comment:
The input topic or topics (multiple topics can be specified as a comma-separated list) of a Pulsar Function